src/Round.js

```diff
@@ -0,0 +1,42 @@
+const Turn = require("./Turn");
+
+class Round {
+    constructor(deck) {
+        this.playingDeck = deck.cards;
+        this.turn;
+        this.turnsCount = 0;
+        this.currentCard = deck.cards[0];
+
+        this.incorrectGuesses = [];
+        this.correctGuesses = [];
+    }
+
+    returnCurrentCard() {
```

Comment:
I feel like, I am going to need to adjust this method to update after the take turn, because I am returning it, and so I don't know if it's re-assigning as a property in my constructor.